index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset = "UTF-8">
+    <meta http-equiv = "X-UA-Compatible" content = "IE=edge">
+    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href="index.css">
+    <link rel = "icon" href="images/favicon.ico" type = "image/x-icon">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+    <title>Calendar Project</title>
+</head>
+<body>
+    <div class = "logo">
```

Comment:
일반적으로 html에서 속성을 정해줄때는 띄어쓰기를 하지 않습니다.
```suggestion
    <div class="logo">
```